scene/resources/default_theme/source/icon_mini_checkerboard.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="16" height="16" version="1.1" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
```

Comment:
this icon is currently in master, so no need to include it.

Reply:
It's the same icon for the scene default_theme

Reply:
oh, i see, you dont have to include it here then.